doc/content/3.documentation/1.concepts/testing.md

```diff
@@ -161,6 +161,46 @@ public async Task ASampleTest()
 }
 ```
 
+### Request / Response
+
+To test a Request using the MassTransit Test Harness:
+
+```csharp
+[Test]
+public async Task ASampleTest()
+{
+    await using var provider = new ServiceCollection()
+        .AddMassTransitTestHarness(cfg =>
+        {
+            cfg.Handler<SubmitOrderConsumer>(async cxt => 
+            {
+                await cxt.RespondAsync(new OrderResponse("OK"));
+            }).Endpoint(ep => ep.Name = "ANamedEndpoint");
+        })
+        .BuildServiceProvider(true);
+
+    var harness = provider.GetRequiredService<ITestHarness>();
+
+    await harness.Start();
+
+    // your actual test code may include a call like below some where
+    var clientFactory = provider.GetRequiredService<IScopedClientFactory>();
+
+    // A Short Address (link below)
+    var serviceAddress = new Uri("queue:ANamedEndpoint");
+    var client = clientFactory.CreateRequestClient<OrderSubmitted>(serviceAddress);
```

Comment:
I get what you're trying to show here, but these seems more useful in the request page vs testing. For testing, it isn't really advised to test routing, you're testing the consumer.

Reply:
I could do one w/o the dedicated endpoint name, but it tends to hide how it all works. But I can pull it out. I would have missed the dedicated endpoint concept when looking for testing. something to think about.